### ISSUE TITLE
A bug in the dataset of CoqGym

### ISSUE THREAD

**Issue by zhaoyu-li** [princeton-vl/CoqGym#61]
Hi, 

I find there is a bug in the dataset that will affect training. In `agent.py`, the `qualid` attribute for const is the full name of the const, (e.g., Coq.Arith.PeanoNat.Nat.mul_wd), however, it should be the `short_ident` (PeanoNat.Nat.mul_wd), since the extracted tactic arguments from the Coq files use the short name, not the full name. This problem will cause that there is no ground-truth label for tactic arguments when training.

And may I ask if the provided checkpoint of ASTacitc trained on only the training set or both training set & testing set, and is the epoch 4 or 5 (since the default epoch is 4 but the paper states that the epoch is 5)?

**Comment by yangky11** [comment]
Hi Zhaoyu,

Thanks for reporting the problem. I haven't been able to take a detailed look yet. I'll probably do it this weekend.

As for the number of epochs, it's probably 5 but we didn't observe differences after epoch 2.

**Comment by zhaoyu-li** [comment]
Hi Kaiyu,

Thanks for your quick response!

**Comment by yangky11** [comment]
Hi Zhaoyu, 

We can leave this issue open if it hasn't been fully resolved. Sorry I'm currently working for the ICLR deadline tomorrow and will have time to take a look later.

**Comment by yangky11** [comment]
It looks like `short_ident ` should be used instead of the full name. A straightforward fix would be to trim the full name. Thanks for reporting the problem!

**Comment by zhaoyu-li** [comment]
Yeah, I just replace the name. Thanks for your quick response and fixing!
